packages/opentelemetry-cloud-monitoring-exporter/test/monitoring.test.ts

```diff
@@ -170,136 +209,169 @@ describe('MetricExporter', () => {
       assert.strictEqual(unhandledPromiseRejectionEvent, false);
     });
 
-    it('should export metrics', async () => {
-      const resourceMetrics = await generateMetricsData();
-      const result = await new Promise<ExportResult>(resolve => {
+    it('should call Monitoring.projects.metricDescriptors.get() with the correct data', async () => {
+      await new Promise<ExportResult>(resolve => {
         exporter.export(resourceMetrics, result => {
           resolve(result);
         });
       });
-      assert.deepStrictEqual(result, {code: ExportResultCode.SUCCESS});
+
       assert.deepStrictEqual(
-        metricDescriptors.getCall(0).args[0].requestBody!.type,
-        'workload.googleapis.com/name'
+        metricDescriptorsGet.getCall(0).args[0].name,
+        `projects/${process.env.GCLOUD_PROJECT}/metricDescriptors/workload.googleapis.com/name`
```

Comment:
nit I'd prefer to just hardcode the fake project id in the string for readability
```suggestion
        `projects/not-real/metricDescriptors/workload.googleapis.com/name`
```

Reply:
Sure, addressed @aabmass.